modules/plugins/randompictures.go

```diff
@@ -270,6 +271,7 @@ func (rp *RandomPictures) Action(command string, content string, msg *discordgo.
 		cursor, err := rethink.Table("randompictures_sources").Run(helpers.GetDB())
 		helpers.Relax(err)
 		err = cursor.All(&rpSources)
+		// TODO: Need to handling this correctly.
```

Comment:
I think similar problem is also exists here.
However I'm not sure how to fix it.
I leave this to you :)

Reply:
Aye will do that!